release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -64,9 +64,139 @@ The {product-title} installation program for Microsoft Azure now creates subnets
 [id="ocp-4-9-ibm-z"]
 === IBM Z and LinuxONE
 
+With this release, IBM Z and LinuxONE are now compatible with {product-title} {product-version}. The installation can be performed with z/VM or {op-system-base} KVM. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network]
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on IBM Z and LinuxONE in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on IBM Z and LinuxONE with {product-title} {product-version}:
+
+* Helm 
+* Support for multiple network interfaces
+* Service Binding Operator
+
+[discrete]
+==== Supported features
+
+The following features are also supported on IBM Z and LinuxONE:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** NFD Operator
+** OpenShift Elasticsearch Operator
+** Local Storage Operator
+** Service Binding Operator
+
+* Encrypting data stored in etcd
+* Multipathing
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* OVN-Kubernetes
+* Three-node cluster support
+* z/VM Emulated FBA devices on SCSI disks
+* 4K FCP block device
+
+These features are available only for {product-title} on IBM Z for {product-version}:
+
+* HyperPAV enabled on IBM Z /LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+==== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
```

Comment:
Is this "IBM Z" going to be clear enough? I might edit this line to be similar to 119 - I think that line is clear in this context without repeating the product names.

Reply:
@ktania46  please change as follows: 

* The following {product-title} Technology Preview features are unsupported: